### PR TITLE
feat: GameObject.GetLocalIdentifierInFile

### PR DESCRIPTION
**Pull request by stan-osipov**
## Purpose of this PR
GameObject.GetLocalIdentifierInFile  extesions method added.

### CODE REVIEW

File: Runtime/Extensions/UnityEngine/GameObjectExtensions.cs [hunk 1]
```diff
@@ -18,6 +19,24 @@ public static void SetLayerRecursively(this GameObject gameObject, int layerNumb
                 trans.gameObject.layer = layerNumber;
         }
         
+        /// <summary>
+        /// Gets the local Identifier In File, for the given GameObject
+        /// Return 0 in case Game Object wasn't yet saved
+        /// </summary>
+        /// <param name="go">GameObject tou want to check</param>
```
Comment (PavelKlym) [inline Runtime/Extensions/UnityEngine/GameObjectExtensions.cs:26]:
```suggestion
        /// <param name="go">GameObject you want to check</param>
```

**Review by PavelKlym — CHANGES_REQUESTED** [review]
Everything cool, thanks for the insanely useful extension method!
Minor spelling bug detected.

And I thought about the same method for Component/MonoBehaviour maybe?

**Comment by stan-osipov** [comment]
Yep make sense, will do updates shortly :)

**Review by PavelKlym — APPROVED** [review]
Great foundation updates, thanks!

**Pull request merged**